python/lsst/ts/cRIOpy/GUI/InclinometerPageWidget.py

```diff
@@ -7,88 +7,44 @@
 class InclinometerPageWidget(QWidget):
     def __init__(self, m1m3):
         super().__init__()
-        self.m1m3 = m1m3
 
-        self.layout = QVBoxLayout()
-        self.dataLayout = QGridLayout()
-        self.warningLayout = QGridLayout()
-        self.layout.addLayout(self.dataLayout)
-        self.layout.addWidget(QLabel(" "))
-        self.layout.addLayout(self.warningLayout)
+        dataLayout = QGridLayout()
 
         self.angleLabel = QLabel("UNKNOWN")
 
-        self.anyWarningLabel = QLabel("UNKNOWN")
-        self.sensorReportsIllegalFunctionLabel = QLabel("UNKNOWN")
-        self.sensorReportsIllegalDataAddressLabel = QLabel("UNKNOWN")
-        self.responseTimeoutLabel = QLabel("UNKNOWN")
-        self.invalidCRCLabel = QLabel("UNKNOWN")
-        self.invalidLengthLabel = QLabel("UNKNOWN")
-        self.unknownAddressLabel = QLabel("UNKNOWN")
-        self.unknownFunctionLabel = QLabel("UNKNOWN")
-        self.unknownProblemLabel = QLabel("UNKNOWN")
-
         self.chart = TimeChart({"Angle (deg)": ["Inclinometer Angle"]})
-        self.chart_view = TimeChartView(self.chart)
-
-        row = 0
-        col = 0
-        self.dataLayout.addWidget(QLabel("Angle (deg)"), row, col)
-        self.dataLayout.addWidget(self.angleLabel, row, col + 1)
 
         row = 0
         col = 0
-        self.warningLayout.addWidget(QLabel("Any Warnings"), row, col)
-        self.warningLayout.addWidget(self.anyWarningLabel, row, col + 1)
-        row += 1
-        self.warningLayout.addWidget(QLabel("Sensor Illegal Function"), row, col)
-        self.warningLayout.addWidget(
-            self.sensorReportsIllegalFunctionLabel, row, col + 1
+        dataLayout.addWidget(QLabel("Angle (deg)"), row, col)
+        dataLayout.addWidget(self.angleLabel, row, col + 1)
+
+        layout = QVBoxLayout()
+        layout.addLayout(dataLayout)
+        layout.addSpacing(20)
+        layout.addWidget(
+            WarningGrid(
+                {
+                    "anyWarning": "Any Warnings",
+                    "sensorReportsIllegalFunctionLabel": "Sensor Illegal Function",
+                    "sensorReportsIllegalAddressLabel": "Sensor Illegal Address",
+                    "responseTimeout": "Reponse Timeout",
+                    "invalidCRC": "Invalid CRC",
```

Comment:
Please consider to add the comment to explain what is full name of `CRC`. No idea of this.

Reply:
CRC is Cyclic Redundancy Check. 
<img width="687" alt="image" src="https://user-images.githubusercontent.com/5473542/139723108-6069c0e4-d053-43f3-9356-592e416b6ba5.png">